### ISSUE TITLE
Correct way of deleting old remote backups

### ISSUE THREAD

**Issue by riskingh** [Altinity/clickhouse-backup#650]
Hi. I am using clickhouse-operator and I used this doc to set up backups: 
https://github.com/AlexAkulov/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes

The problem is that `clickhouse-backup server` does not seem to respect `backups_to_keep_remote: 3` and all backups are kept. What is the correct way of deleting them? Do I need to use `watch` command instead?

**Comment by Slach** [comment]
could you share
`clickhouse-backup list remote` command?

Old backup will delete if no one other backup is not reference to this as "required"

**Comment by riskingh** [comment]
here it is:
```bash-5.1$ LOG_LEVEL=error clickhouse-backup list remote 2>/dev/null
v4-cluster-s0-r0-full-2023-02-15-10-48-51   224.24GiB   15/02/2023 11:45:14   remote      tar, regular
v4-cluster-s1-r0-full-2023-02-15-10-48-51   238.48GiB   15/02/2023 11:56:41   remote      tar, regular
v4-cluster-s0-r0-full-2023-02-16-00-00-03   233.73GiB   16/02/2023 01:12:05   remote      tar, regular
v4-cluster-s1-r0-full-2023-02-16-00-00-03   248.79GiB   16/02/2023 01:15:05   remote      tar, regular
v4-cluster-s0-r1-full-2023-02-17-12-55-30   260.30GiB   17/02/2023 13:48:42   remote      tar, regular
v4-cluster-s1-r1-full-2023-02-17-12-55-30   278.20GiB   17/02/2023 13:54:45   remote      tar, regular
v4-cluster-s0-r1-full-2023-02-28-14-24-52   384.21GiB   28/02/2023 15:56:07   remote      tar, regular
v4-cluster-s1-r1-full-2023-02-28-14-24-52   406.84GiB   28/02/2023 16:02:48   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-01-00-00-07   388.22GiB   01/03/2023 01:36:47   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-01-00-00-07   410.82GiB   01/03/2023 01:42:22   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-02-00-00-29   398.78GiB   02/03/2023 01:38:02   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-02-00-00-29   421.50GiB   02/03/2023 01:45:05   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-03-00-00-03   409.27GiB   03/03/2023 01:39:45   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-03-00-00-03   432.46GiB   03/03/2023 01:45:07   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-04-00-00-04   421.32GiB   04/03/2023 01:35:43   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-04-00-00-04   444.43GiB   04/03/2023 01:41:11   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-05-00-00-03   430.87GiB   05/03/2023 01:31:28   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-05-00-00-03   452.89GiB   05/03/2023 01:38:50   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-06-00-00-04   433.56GiB   06/03/2023 01:31:51   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-06-00-00-04   455.89GiB   06/03/2023 01:38:15   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-07-00-00-04   435.19GiB   07/03/2023 01:38:37   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-07-00-00-04   457.16GiB   07/03/2023 01:44:54   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-08-00-00-04   436.44GiB   08/03/2023 01:40:36   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-08-00-00-04   458.12GiB   08/03/2023 01:46:03   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-09-00-00-26   437.79GiB   09/03/2023 01:43:59   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-09-00-00-26   459.54GiB   09/03/2023 01:49:58   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-10-00-00-28   438.92GiB   10/03/2023 01:42:31   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-10-00-00-28   460.31GiB   10/03/2023 01:49:40   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-11-00-00-03   440.25GiB   11/03/2023 01:41:06   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-11-00-00-03   461.73GiB   11/03/2023 01:46:28   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-12-00-00-22   441.29GiB   12/03/2023 01:34:27   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-12-00-00-22   459.27GiB   12/03/2023 01:39:10   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-13-00-00-03   442.68GiB   13/03/2023 01:30:13   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-13-00-00-03   460.82GiB   13/03/2023 01:34:57   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-14-00-00-03   443.85GiB   14/03/2023 01:34:06   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-14-00-00-03   461.82GiB   14/03/2023 01:38:47   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-15-00-00-04   444.80GiB   15/03/2023 01:35:40   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-15-00-00-04   462.19GiB   15/03/2023 01:40:13   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-16-00-00-06   445.74GiB   16/03/2023 01:37:19   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-16-00-00-06   463.57GiB   16/03/2023 01:42:37   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-17-00-00-03   447.05GiB   17/03/2023 01:37:42   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-17-00-00-03   463.68GiB   17/03/2023 01:42:50   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-18-00-00-03   446.47GiB   18/03/2023 01:35:11   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-18-00-00-03   463.85GiB   18/03/2023 01:38:58   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-19-00-00-05   446.14GiB   19/03/2023 01:29:58   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-19-00-00-05   464.29GiB   19/03/2023 01:34:39   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-20-00-00-04   447.01GiB   20/03/2023 01:30:28   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-20-00-00-04   465.73GiB   20/03/2023 01:35:12   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-21-00-00-03   448.47GiB   21/03/2023 01:36:46   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-21-00-00-03   467.37GiB   21/03/2023 01:41:53   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-22-00-00-06   449.88GiB   22/03/2023 01:38:42   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-22-00-00-06   468.44GiB   22/03/2023 01:42:41   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-23-00-00-04   451.68GiB   23/03/2023 01:38:32   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-23-00-00-04   470.31GiB   23/03/2023 01:44:29   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-24-00-00-04   453.33GiB   24/03/2023 01:38:40   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-24-00-00-04   472.38GiB   24/03/2023 01:44:05   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-25-00-00-04   455.05GiB   25/03/2023 01:35:23   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-25-00-00-04   474.27GiB   25/03/2023 01:40:44   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-26-00-00-04   455.85GiB   26/03/2023 01:31:35   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-26-00-00-04   475.87GiB   26/03/2023 01:36:50   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-27-00-00-03   457.29GiB   27/03/2023 01:33:11   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-27-00-00-03   477.40GiB   27/03/2023 01:39:10   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-28-00-00-04   458.96GiB   28/03/2023 01:38:18   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-28-00-00-04   479.14GiB   28/03/2023 01:43:39   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-29-00-00-25   460.63GiB   29/03/2023 01:38:44   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-29-00-00-25   480.93GiB   29/03/2023 01:44:35   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-30-00-00-03   462.40GiB   30/03/2023 01:41:37   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-30-00-00-03   482.69GiB   30/03/2023 01:47:09   remote      tar, regular
v4-cluster-s0-r1-full-2023-03-31-00-00-04   464.00GiB   31/03/2023 01:42:58   remote      tar, regular
v4-cluster-s1-r1-full-2023-03-31-00-00-04   484.00GiB   31/03/2023 01:49:14   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-01-00-00-03   465.06GiB   01/04/2023 01:41:52   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-01-00-00-03   485.43GiB   01/04/2023 01:47:50   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-02-00-00-03   466.67GiB   02/04/2023 01:39:04   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-02-00-00-03   487.06GiB   02/04/2023 01:44:35   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-03-00-00-04   467.83GiB   03/04/2023 01:34:25   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-03-00-00-04   488.05GiB   03/04/2023 01:41:02   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-04-00-00-04   469.50GiB   04/04/2023 01:42:22   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-04-00-00-04   489.15GiB   04/04/2023 01:49:09   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-05-00-00-04   470.40GiB   05/04/2023 01:43:37   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-05-00-00-04   490.79GiB   05/04/2023 01:49:53   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-06-00-00-03   471.97GiB   06/04/2023 01:46:11   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-06-00-00-03   492.62GiB   06/04/2023 01:54:13   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-07-00-00-04   473.68GiB   07/04/2023 01:45:48   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-07-00-00-04   494.44GiB   07/04/2023 01:52:16   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-08-00-00-03   475.38GiB   08/04/2023 01:42:08   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-08-00-00-03   496.12GiB   08/04/2023 01:48:34   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-09-00-00-03   476.80GiB   09/04/2023 01:37:35   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-09-00-00-03   497.85GiB   09/04/2023 01:43:53   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-10-00-00-03   478.50GiB   10/04/2023 01:37:55   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-10-00-00-03   499.42GiB   10/04/2023 01:44:11   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-11-00-00-03   480.16GiB   11/04/2023 01:46:08   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-11-00-00-03   501.20GiB   11/04/2023 01:52:48   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-12-00-00-04   482.38GiB   12/04/2023 01:48:05   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-12-00-00-04   503.49GiB   12/04/2023 01:56:19   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-13-00-00-03   484.10GiB   13/04/2023 01:47:27   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-13-00-00-03   505.27GiB   13/04/2023 01:54:43   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-14-00-00-04   485.83GiB   14/04/2023 01:47:12   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-14-00-00-04   507.06GiB   14/04/2023 01:54:46   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-15-00-00-04   487.63GiB   15/04/2023 01:45:25   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-15-00-00-04   508.94GiB   15/04/2023 01:53:04   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-16-00-00-03   489.34GiB   16/04/2023 01:41:02   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-16-00-00-03   510.62GiB   16/04/2023 01:47:16   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-17-00-00-03   490.89GiB   17/04/2023 01:40:12   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-17-00-00-03   512.28GiB   17/04/2023 01:48:29   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-18-00-00-04   492.76GiB   18/04/2023 01:46:48   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-18-00-00-04   514.19GiB   18/04/2023 01:54:40   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-19-00-00-08   494.62GiB   19/04/2023 01:49:00   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-19-00-00-08   516.12GiB   19/04/2023 01:55:58   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-20-00-00-04   496.49GiB   20/04/2023 01:48:08   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-20-00-00-04   517.84GiB   20/04/2023 01:56:17   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-21-00-00-03   498.20GiB   21/04/2023 01:48:53   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-21-00-00-03   519.58GiB   21/04/2023 01:56:38   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-22-00-00-03   499.95GiB   22/04/2023 01:47:50   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-22-00-00-03   521.31GiB   22/04/2023 01:55:12   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-23-00-00-03   501.45GiB   23/04/2023 01:42:01   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-23-00-00-03   523.00GiB   23/04/2023 01:48:10   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-24-00-00-04   502.95GiB   24/04/2023 01:41:43   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-24-00-00-04   524.55GiB   24/04/2023 01:48:39   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-25-00-00-04   504.67GiB   25/04/2023 01:46:31   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-25-00-00-04   526.30GiB   25/04/2023 01:54:01   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-26-00-00-06   506.42GiB   26/04/2023 01:47:18   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-26-00-00-06   528.11GiB   26/04/2023 01:54:20   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-27-00-00-06   508.13GiB   27/04/2023 01:46:16   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-27-00-00-06   529.77GiB   27/04/2023 01:52:37   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-28-00-00-04   509.89GiB   28/04/2023 01:49:44   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-28-00-00-04   531.53GiB   28/04/2023 01:57:25   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-29-00-00-03   511.53GiB   29/04/2023 01:48:19   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-29-00-00-03   533.23GiB   29/04/2023 01:54:51   remote      tar, regular
v4-cluster-s0-r1-full-2023-04-30-00-00-03   513.12GiB   30/04/2023 01:45:36   remote      tar, regular
v4-cluster-s1-r1-full-2023-04-30-00-00-03   534.83GiB   30/04/2023 01:52:28   remote      tar, regular
v4-cluster-s0-r1-full-2023-05-01-00-00-04   514.86GiB   01/05/2023 01:43:45   remote      tar, regular
v4-cluster-s1-r1-full-2023-05-01-00-00-04   535.98GiB   01/05/2023 01:50:57   remote      tar, regular
v4-cluster-s0-r1-full-2023-05-02-00-00-03   516.30GiB   02/05/2023 01:48:50   remote      tar, regular
v4-cluster-s1-r1-full-2023-05-02-00-00-03   537.78GiB   02/05/2023 01:56:57   remote      tar, regular
v4-cluster-s0-r1-full-2023-05-03-00-00-04   518.16GiB   03/05/2023 01:53:03   remote      tar, regular
v4-cluster-s1-r1-full-2023-05-03-00-00-04   539.51GiB   03/05/2023 02:00:42   remote      tar, regular
```

**Comment by Slach** [comment]
could you share
`clichouse-backup print-config` without sensitive credentials?

**Comment by riskingh** [comment]
```general:
  remote_storage: s3
  max_file_size: 0
  disable_progress_bar: true
  backups_to_keep_local: 2
  backups_to_keep_remote: 3
  log_level: debug
  allow_empty_backups: true
  download_concurrency: 2
  upload_concurrency: 2
  use_resumable_state: false
  restore_schema_on_cluster: ""
  upload_by_part: true
  download_by_part: true
  restore_database_mapping: {}
  retries_on_failure: 3
  upload_retries_pause: 30s
  watch_interval: 1h
  full_interval: 24h
  watch_backup_name_template: shard{shard}-{type}-{time:20060102150405}
  retriesduration: 30s
  watchduration: 1h0m0s
  fullduration: 24h0m0s
clickhouse:
  username: <username>
  password: <password>
  host: localhost
  port: 9000
  disk_mapping: {}
  skip_tables:
  - system.*
  - INFORMATION_SCHEMA.*
  - information_schema.*
  - _temporary_and_external_tables.*
  timeout: 5m
  freeze_by_part: false
  freeze_by_part_where: ""
  use_embedded_backup_restore: false
  embedded_backup_disk: ""
  secure: false
  skip_verify: false
  sync_replicated_tables: false
  log_sql_queries: true
  config_dir: /etc/clickhouse-server/
  restart_command: systemctl restart clickhouse-server
  ignore_not_exists_error_during_freeze: true
  check_replicas_before_attach: true
  tls_key: ""
  tls_cert: ""
  tls_ca: ""
  debug: false
s3:
  access_key: <aws_key>
  secret_key: <secret_key>
  bucket: <bucket>
  endpoint: ""
  region: <region>
  acl: private
  assume_role_arn: ""
  force_path_style: false
  path: ""
  disable_ssl: false
  compression_level: 1
  compression_format: tar
  sse: ""
  disable_cert_verification: false
  use_custom_storage_class: false
  storage_class: STANDARD
  concurrency: 1
  part_size: 0
  max_parts_count: 10000
  allow_multipart_download: false
  object_labels: {}
  debug: true
gcs:
  credentials_file: ""
  credentials_json: ""
  credentials_json_encoded: ""
  bucket: ""
  path: ""
  compression_level: 1
  compression_format: tar
  debug: false
  endpoint: ""
  storage_class: STANDARD
  object_labels: {}
cos:
  url: ""
  timeout: 2m
  secret_id: ""
  secret_key: ""
  path: ""
  compression_format: tar
  compression_level: 1
  debug: false
api:
  listen: 0.0.0.0:7171
  enable_metrics: true
  enable_pprof: false
  username: ""
  password: ""
  secure: false
  certificate_file: ""
  private_key_file: ""
  create_integration_tables: true
  integration_tables_host: ""
  allow_parallel: false
ftp:
  address: ""
  timeout: 2m
  username: ""
  password: ""
  tls: false
  path: ""
  compression_format: tar
  compression_level: 1
  concurrency: 2
  debug: false
sftp:
  address: ""
  port: 22
  username: ""
  password: ""
  key: ""
  path: ""
  compression_format: tar
  compression_level: 1
  concurrency: 1
  debug: false
azblob:
  endpoint_schema: https
  endpoint_suffix: core.windows.net
  account_name: ""
  account_key: ""
  sas: ""
  use_managed_identity: false
  container: ""
  path: ""
  compression_level: 1
  compression_format: tar
  sse_key: ""
  buffer_size: 0
  buffer_count: 3
  max_parts_count: 10000
  timeout: 15m
custom:
  upload_command: ""
  download_command: ""
  list_command: ""
  delete_command: ""
  command_timeout: 4h
  commandtimeoutduration: 4h0m0s
```

**Comment by riskingh** [comment]
```bash-5.1$ clickhouse-backup --version
Version:         d150ec7b9330b2d93031334c2624df424759df1f
Git Commit:      d150ec7b9330b2d93031334c2624df424759df1f
Build Date:      2023-02-07
```

**Comment by Slach** [comment]
Do you have `RemoveOldBakups` string in your logs during execute upload or create_remote?

**Comment by riskingh** [comment]
Can't find such string. Only see this:
```
2023/02/28 05:35:44.406332  info Update backup metrics finish LastBackupCreateLocal=2023-02-16 00:00:08.007054854 +0000 UTC LastBackupCreateRemote=2023-02-17 12:55:35.150776579 +0000 UTC LastBackupSizeLocal=250944219494 LastBackupSizeRemote=298692320965 LastBackupUpload=2023-02-17 13:54:45 +0000 UTC NumberBackupsLocal=2 NumberBackupsRemote=260 duration=560ms logger=server
```

I also see 403 to `GET /?versioning= HTTP/1.1` request:
```
2023/02/28 05:35:44.086650  info [s3:DEBUG] request failed with unretryable error https response error StatusCode: 403, RequestID: ..., HostID: ...., api error AccessDenied: Access Denied
2023/02/28 05:35:44.086726 debug /tmp/.clickhouse-backup-metadata.cache.S3 not found, load 0 elements logger=s3
```

**Comment by Slach** [comment]
`LastBackupCreateRemote=2023-02-17 12:55:35.150776579`
looks weird, it shall be fixed on 2.2.5
could you upgrade to latest 2.2.5 release?
and check again?

**Comment by riskingh** [comment]
Oh, my bad, found actual logs and the problem is that I didn't give s3 user delete permissions. Thanks!
